service/iot-service-client/pom.xml

```diff
@@ -32,8 +32,32 @@
     </scm>
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <ms-rest-client-version>1.7.9</ms-rest-client-version>
     </properties>
+    <repositories>
```

Comment:
This is required to generate private jars and will be removed before checking in.